src/utils/amplitude.ts

```diff
@@ -193,8 +193,8 @@ export function generateTransferQuote(
     relayGasFeeTotal: formatTokens(fees.relayerGasFee.total),
     relayGasFeeTotalUsd: usdEquivalentString(fees.relayerGasFee.total),
     sender: account,
-    routeChainIdFromTo: toChainInfo.chainId.toString(),
-    routeChainNameFromTo: toChainInfo.name,
+    routeChainIdFromTo: `${fromChainInfo.chainId} -> ${toChainInfo.chainId}`,
+    routeChainNameFromTo: `${fromChainInfo.name} -> ${toChainInfo.name}`,
```

Comment:
For the server-side event where we also have these properties, we use the format
```suggestion
    routeChainIdFromTo: `${fromChainInfo.chainId}-${toChainInfo.chainId}`,
    routeChainNameFromTo: `${fromChainInfo.name}-${toChainInfo.name}`,
```
I guess we should stay consistent